ksql-engine/src/test/java/io/confluent/ksql/function/udf/json/ArrayContainsKudfTest.java

```diff
@@ -0,0 +1,144 @@
+package io.confluent.ksql.function.udf.json;
+
+import junit.framework.TestCase;
+import org.junit.Test;
+
+public class ArrayContainsKudfTest
+        extends TestCase
+{
+    private ArrayContainsKudf jsonUdf = new ArrayContainsKudf();
+
+    @Override
+    public void setUp()
+            throws Exception
+    {
+        jsonUdf.init();
+    }
+
+    @Test
+    public void testEmptyArray()
+    {
+        assertEquals(false, jsonUdf.evaluate("[]", true));
+        assertEquals(false, jsonUdf.evaluate("[]", false));
+        assertEquals(false, jsonUdf.evaluate("[]", null));
+        assertEquals(false, jsonUdf.evaluate("[]", 1.0));
+        assertEquals(false, jsonUdf.evaluate("[]", 100));
+        assertEquals(false, jsonUdf.evaluate("[]", "abc"));
+        assertEquals(false, jsonUdf.evaluate("[]", ""));
+    }
+
+    @Test
+    public void testNullArray()
```

Comment:
split this into two tests, i.e., one for when it should be true and on for when it should be false. Also naming as proposed above